src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -44,25 +44,48 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2
 
+_INITIAL_METADATA_KEY = "x-grpc-test-echo-initial"
+_TRAILING_METADATA_KEY = "x-grpc-test-echo-trailing-bin"
+
+# If the request came with metadata in the context, this
+# helper function will copy it to the returned metadata.
+def _maybe_echo_metadata(context):
+  metadata = dict(context.invocation_metadata())
+  if _INITIAL_METADATA_KEY in metadata:
+    initial_metadata = (
```

Comment:
In orthodox gRPC vocabulary, this single key-value pair is a metadatum, not metadata.